jsBasics/task1.js

```diff
@@ -0,0 +1,23 @@
+const title = 'Welcome to Richmond! '
+
+console.log(title)
+
+const SUPER_ADMINT_EMAIL = 'aa@mail.com'
+
+console.log(SUPER_ADMINT_EMAIL)
+
+const year = 1995
+
+console.log('Richmond till we die since ' + year)
+
+const hasWon = true
+
+console.log(hasWon)
+
+const player = null
+const coach1 = undefined
+const coach2 
```

Comment:
no, you can't do it this way. Value should be assigned if you use const. In this case you should use let if you do not want to assign any value while declaring variable